evalml/automl/automl_search.py

```diff
@@ -289,6 +289,10 @@ def __init__(self,
             allowed_estimators = get_estimators(self.problem_type, self.allowed_model_families)
             logger.debug(f"allowed_estimators set to {[estimator.name for estimator in allowed_estimators]}")
             self.allowed_pipelines = [make_pipeline(self.X_train, self.y_train, estimator, self.problem_type, custom_hyperparameters=self.pipeline_parameters) for estimator in allowed_estimators]
+        else:
```

Comment:
This is to fix #2132. Without this change the hyperparameters passed in to ``pipeline_parameters`` would only get used in the first batch.